src/main/java/com/unstoppabledomains/resolution/Resolution.java

```diff
@@ -93,6 +93,17 @@ public String getAddress(String domain, String ticker) throws NamingServiceExcep
         }
     }
 
+    @Override
+    public String getMultiChainAddress(String domain, String ticker, String chain) throws NamingServiceException {
+        NamingService service = findService(domain);
+        if (service.getType().equals(NamingServiceType.ENS)) {
```

Comment:
You can simply compare enums with `==`:
```suggestion
        if (service.getType() == NamingServiceType.ENS) {
```